index.js

```diff
@@ -0,0 +1,158 @@
+import * as core from "@actions/core";
+import { LinearClient } from "@linear/sdk";
+
+const ESCAPE = {
+  ">": "&gt;",
+  "<": "&lt;",
+  "&": "&amp;",
+  "\r\n": " ",
+  "\n": " ",
+  "\r": " ",
+};
+const ESPACE_REGEX = new RegExp(Object.keys(ESCAPE).join("|"), "gi");
+
+const getBugs = async (linearClient, stateIds, label) => {
+  const issues = await linearClient.issues({
+    filter: {
+      labels: {
+        and: [{ name: { eq: "Bug" } }, label ? { name: { eq: label } } : {}],
+      },
+      state: { id: { in: stateIds } },
+      project: { null: true },
+    },
+  });
+
+  return removeChildIssues(issues);
+};
+
+const getChores = async (linearClient, stateIds, label) => {
+  const issues = await linearClient.issues({
+    filter: {
+      labels: {
+        and: [{ name: { eq: "Chore" } }, label ? { name: { eq: label } } : {}],
+      },
+      state: { id: { in: stateIds } },
+      project: { null: true },
+    },
+  });
+
+  return removeChildIssues(issues);
+};
+
+const getFastTracks = async (linearClient, stateIds, label) => {
+  const issues = await linearClient.issues({
+    filter: {
+      labels: {
+        and: [
+          { every: { name: { nin: ["Bug", "Chore"] } } },
```

Comment:
we have a label "fast track", why not use it?

Reply:
Could be that a task is missing a label. Then it goes into fast track 🤷🏻‍♂️

Reply:
would be better to have a category "other" for such cases

Reply:
@jmorrice What do you think?

Reply:
Yes could be nice to have something like that, currently quite a few issues are incorrectly listed under Fast Track. That's not really what this PR is about though :D 

<img width="615" alt="CleanShot 2023-01-10 at 09 12 30@2x" src="https://user-images.githubusercontent.com/5297632/211496564-fe40c44c-81ea-45e1-9f68-80aa5931482f.png">

Reply:
👌🏻